gcp/appengine/frontend_handlers.py

```diff
@@ -366,7 +367,7 @@ def osv_query(search_string, page, affected_only, ecosystem):
     query = query.filter(osv.Bug.search_indices == search_string.lower())
 
   if affected_only:
-    query = query.filter(osv.Bug.has_affected == True)  # pylint: disable=singleton-comparison
+    query = query.filter(osv.Bug.has_affected is True)  # pylint: disable=singleton-comparison
```

Comment:
does this actually work with ndb? 

I don't think this worked in the past, unless a more recent version of ndb has changed this.

Reply:
No, I blindly acted on a warning VS Code spat out, my bad. Fixed in [45432a6](https://github.com/google/osv.dev/pull/2031/commits/45432a66c566af0072022832cb3a526e56d9be22)